charts/keycloakx/ci/h2-values.yaml

```diff
@@ -20,6 +22,10 @@ extraEnv: |
       -XX:MaxRAMPercentage=50.0
       -Djava.awt.headless=true
       -Djgroups.dns.query={{ include "keycloak.fullname" . }}-headless
+  - name: KC_HOSTNAME
+    value: keycloak-ci
+  - name: KC_FEATURES_DISABLED
+    value: admin2
```

Comment:
```suggestion
```

Testing the new admin console with the Chrome driver now passes, so I think it's better to test with default behavior.

Reply:
https://github.com/codecentric/helm-charts/blob/cab3997511239923a030333e006db569239d2ca3/charts/keycloakx/README.md?plain=1#L11-L25

I think it's more reliable to define the test manifest according to the guidance in the `README`.